src/Dapr.PluggableComponents/AssemblyInfo.cs

```diff
@@ -1,4 +1,4 @@
-// ------------------------------------------------------------------------
+﻿// ------------------------------------------------------------------------
```

Comment:
Apparently GitHub can't handle trailing whitespace in files with MS-DOS/Windows line-endings??